### ISSUE TITLE
Build failure with GHC 7.4.2 and tasty-0.11.2.4

### ISSUE THREAD

**Issue by grayjay** [feuerbach/tasty#181]
The bounds allow base-4.5.1.0, but the cabal GHC 7.4.2 build failed with

```
[16 of 18] Compiling Test.Tasty.CmdLine ( Test/Tasty/CmdLine.hs, dist/build/Test/Tasty/CmdLine.o )

Test/Tasty/CmdLine.hs:20:28:
    Module `Control.Concurrent' does not export `mkWeakThreadId'
cabal: Failed to build tasty-0.11.2.4 (which is required by test:unit-tests
from Cabal-2.1.0.0).
```

Full log: https://travis-ci.org/haskell/cabal/jobs/264167963

**Comment by feuerbach** [comment]
Should be fixed in tasty-0.11.2.5.

**Comment by grayjay** [comment]
Thanks!
